src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -509,6 +511,22 @@ public void testRequireOneOfVCFOrSampleNameFile(){
         runCommandLine(args);
     }
 
+    @Test
+    public void testGenomicsDBImportWithoutDBField() throws IOException {
+	//Test for https://github.com/broadinstitute/gatk/issues/3736
+	final List<String> vcfInputs = Arrays.asList(NA_24385);
+        final String workspace = createTempDir("genomicsdb-tests").getAbsolutePath() + "/workspace";
+	writeToGenomicsDB(vcfInputs, INTERVAL_3736, workspace, 0, false, 0, 1);
+    }
+    
+    @Test
+    public void testLongWorkspacePath() throws IOException {
+	//Test for https://github.com/broadinstitute/gatk/issues/4160
+        final List<String> vcfInputs = LOCAL_GVCFS;
+        final String workspace = createTempDir("long_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa_genomicsdb").getAbsolutePath() + "/should_not_fail_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
```

Comment:
As discussed in person, can you open a separate PR that adds about 150 additional a's to this path @psfoley, and confirm that the test fails if you downgrade the GenomicsDB version in our `build.gradle`? Thanks!